README.md

```diff
@@ -563,9 +563,9 @@ INFO: Successfully synchronised [n] pipelines
 
 If you use `nf-core tools` in your work, please cite the `nf-core` preprint as follows:
 
-> **The nf-core framework for community-curated bioinformatics pipelines**
+> **The nf-core framework for community-curated bioinformatics pipelines.**
 >
-> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
 >
-> [_Nat Biotechnology_ (2020)](https://doi.org/10.1038/s41587-020-0439-x);
-> doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x)
+> _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).  
```

Comment:
Do we need the short version? I think the long version is prettier 🙄 

```suggestion
> _Nature Biotechnology_ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).  
```

Reply:
I quite like the short version and thats how its always listed on PubMed. Not fussed either way to be honest. Shall I change it back?

Reply:
I guess it’s just a personal preference thing. Having a combination of short and long form as I wrote is definitely wrong though!